packages/flutter_reactive_ble/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.0.2
+
+* Revert Queue up messages on iOS until event channel is ready. Fix #439
```

Comment:
Quote the title of the reverted pull request, otherwise the meaning is not very clear.